app/controllers/admin/terms_of_service/histories_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Admin::TermsOfService::HistoriesController < Admin::BaseController
+  def show
```

Comment:
This view is mainly listing these ... could be `index`?

Reply:
But I don't want the URL to be `/histories`, it makes sense as `/history` singular.

Reply:
Fair enough - though the url could be aliased.

Alt idea - `revisions#index`